### ISSUE TITLE
Automatic gradient clipping is not supported for manual optimization

### ISSUE THREAD

**Issue by huguesva** [rbalestr-lab/stable-pretraining#246]
raise MisconfigurationException(
lightning.fabric.utilities.exceptions.MisconfigurationException: Automatic gradient clipping is not supported for manual optimization. Remove `Trainer(gradient_clip_val=1.0)` or switch to automatic optimization.
